examples/interfaces/stan.ipynb

```diff
@@ -0,0 +1,992 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interfacing with Stan\n",
+    "This notebook explains how it is possible to interface with [Stan](https://mc-stan.org/) to make use of their awesome library for log-probability calculations, gradient calculation (via autodiff) and their large zoo of probability distributions.\n",
+    "\n",
+    "One thing to be mindful of is that the interface below only allows the log probability to be accessed from Stan objects up to an additive constant. Another is that Stan operates on the unconstrained scale, so the PINTS interface to Stan passes and receives parameter draws on this scale.\n",
```

Comment:
> the log probability

from the last sentence, it's not clear to me that there is going to be a log probability, so needs a bit more introduction!